tests/test_utils.py

```diff
@@ -79,6 +79,13 @@ def test_make_absolute_href(self):
             actual = make_absolute_href(source_href, start_href)
             self.assertEqual(actual, expected)
 
+    def test_make_absolute_href_on_vsitar(self):
+        cat_path = '/vsitar//tmp/catalog.tar/catalog.json'
+        rel_path = 'some/item.json'
+        result = make_absolute_href(rel_path, cat_path)
```

Comment:
Minor suggestion for clarity:

```
rel_path = 'some/item.json'
cat_path = '/vsitar//tmp/catalog.tar/catalog.json'
expected = '/vsitar//tmp/catalog.tar/some/item.json'

self.assertEqual(expected, make_absolute_href(rel_path, cat_path))
```

Reply:
Ok, thanks. That is minor, but I made the change.

Can you mark PRs that have minor suggestions as approved, and use 'request changes' if there's specific changes or discussion left? Thanks!

I believe this falls into the former category, so I'll merge once CI passes.